contracts/src/v0.8/dev/interfaces/OCR2DRBillableInterface.sol

```diff
@@ -0,0 +1,29 @@
+// SPDX-License-Identifier: MIT
+pragma solidity ^0.8.6;
+
+import "./OCR2DRRegistryInterface.sol";
+
+/**
+ * @title OCR2DR billable oracle interface.
+ */
+interface OCR2DRBillableInterface {
```

Comment:
Could roll this into OCR2DROracleInterface.sol if this is confusing to be separate.

Reply:
Finished.